uwp/Charts/Area.md

```diff
@@ -166,7 +166,7 @@ chart.AreaBorderThickness = new Thickness(3);
 
 {% endtabs %}
 
-![SfChart with customized plot area](Area_images/Area_img3.jpeg)
+![SfChart with customized plot area in UWP](area_images/uwp-charts-area-customized-plot.jpeg)
```

Comment:
we need to give meaningful image name.
Change the image name like as below,
uwp-charts-plot-area-customization

Reply:
Renamed